examples/tutorials/vectors.py

```diff
@@ -0,0 +1,486 @@
+# passed to the plot function and the resulting figure is shown below.
+
+ct = 5
+radius = 3 - (0.5 * np.arange(0, ct))
+startdir = np.full(ct, 90)
+stopdir = 180 + (50 * np.arange(0, ct))
+
+data = np.column_stack([np.full(ct, 0), np.full(ct, 0), radius, startdir, stopdir])
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[-5, 5, -5, 5],
+    projection="X10c",
+    frame="ag",
+    data=data,
+    style="m0.5c+ea",
+    pen="2p",
+    color="red3",
+)
+fig.show()
+
+########################################################################################
+# Much like when plotting regular vectors, the default unit used is centimeters.
+# When this is changed to inches, the size of the plot appears larger when the
+# projection units do not change. Below is an example of two circular vectors.
+# One is plotted using the default unit, and the second is plotted using inches.
+# The difference in size of the two vectors provides good insight into how this
+# functionality works.
+
+circular_vector_1 = [6, 5, 2, 90, 270]
+circular_vector_2 = [6, 5, 1, 90, 270]
+
+data_1 = [circular_vector_1]
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X10c",
+    frame="ag",
+    data=data_1,
+    style="m0.5c+ea",
+    pen="2p",
+    color="red3",
+)
+
+data_2 = [circular_vector_2]
+with pygmt.config(PROJ_LENGTH_UNIT="i"):
+    fig.plot(
+        data=data_2,
+        style="m0.5c+ea",
+        pen="2p",
+        color="red3",
+    )
+fig.show()
+
+########################################################################################
+# Plot Geographic Vectors
+# -----------------------
+# Geographic graph using x and y values to set a start and an ending point.
+# Use ``fig.coast`` to plot a coast. ``x`` and ``y`` are coordinates
+# on a grid that we are using. ``x`` is Idaho and ``y`` is Chicago in this example.
+# The geographical vector is going from Idaho to Chicago. To style geographic
+# vectors, use ``=`` at the begining of the ``style`` parameter.
+# Other styling features such as arrow head color and line thickness
+# can be passed into
+# ``pen`` and ``color``parameters
+
+fig = pygmt.Figure()
+fig.coast(
+    region=[-127, -64, 24, 53],
+    projection="M10c",
+    frame=True,
+    borders=1,
+    shorelines="0.25p,black",
+    area_thresh=4000,
+)
+point_1 = [-114.7420, 44.0682]
+point_2 = [-87.6298, 41.8781]
+data = np.array([point_1 + point_2])
+
+fig.plot(
+    data=data,
+    style="=0.5c+ea+s",
+    pen="2p",
+    color="red3",
+)
+fig.show()
+
+########################################################################################
+# This Geographic Vector is using the `fig.coast` of the region of the United States.
+# The plotting of the geographic vectors when using latitude and longitude
+# are labeled by having the coordinates displayed.
+# Then an array is created so the vectors follow the one vector before it. You
+# can diplay this array any way you want.
```

Comment:
I feel this paragraph difficult to read. Could you please rephrase it?

Reply:
Me, too